directus-solidstart/index.md

```diff
@@ -0,0 +1,314 @@
+---
+title: "Getting Started with Directus and SolidStart"
+description: "Let's learn how to build a blog using Directus as a CMS. You will store, retrieve, and use global metadata such as the site title, create new pages dynamically based on Directus items"
+author:
+  name: "Atila Fassina"
+  avatar_file_name: "author.webp"
+---
+
+## Summary
+
+[SolidStart](https://start.solidjs.com) a meta-framework built around [SolidJS](https://solidjs.com) - SolidStart is a framework aiming to allow people and teams to built high-end applications while bringing their own opinions. In this tutorial, you will learn how to build a website using Directus as a CMS. You will store, retrieve, and use global metadata such as the site title, create new pages dynamically based on Directus items, and build a blog.
```

Comment:
```suggestion
[SolidStart](https://start.solidjs.com) a meta-framework built around [SolidJS](https://solidjs.com) - SolidStart is a framework aiming to allow people and teams to built high-end applications without being too opinionated. In this tutorial, you will learn how to build a website using Directus as a CMS. You will store, retrieve, and use global metadata such as the site title, create new pages dynamically based on Directus items, and build a blog.
```

When you say 'with their own opinions' it doesn't make as much sense as the word 'opinionated' as developers have come to know it.